packages/admin-next/dashboard/src/v2-routes/users/user-invite/components/invite-user-form/invite-user-form.tsx

```diff
@@ -137,64 +140,17 @@ export const InviteUserForm = () => {
-                    <div>
-                      <Table>
-                        <Table.Header className="border-t-0">
-                          {table.getHeaderGroups().map((headerGroup) => {
-                            return (
-                              <Table.Row
-                                key={headerGroup.id}
-                                className="[&_th:last-of-type]:w-[1%] [&_th:last-of-type]:whitespace-nowrap [&_th]:w-1/3"
-                              >
-                                {headerGroup.headers.map((header) => {
-                                  return (
-                                    <Table.HeaderCell key={header.id}>
-                                      {flexRender(
-                                        header.column.columnDef.header,
-                                        header.getContext()
-                                      )}
-                                    </Table.HeaderCell>
-                                  )
-                                })}
-                              </Table.Row>
-                            )
-                          })}
-                        </Table.Header>
-                        <Table.Body>
-                          {table.getRowModel().rows.map((row) => (
-                            <Table.Row
-                              key={row.id}
-                              className={clx(
-                                "transition-fg[&_td:last-of-type]:w-[1%] [&_td:last-of-type]:whitespace-nowrap",
-                                {
-                                  "bg-ui-bg-highlight hover:bg-ui-bg-highlight-hover":
-                                    row.getIsSelected(),
-                                }
-                              )}
-                            >
-                              {row.getVisibleCells().map((cell) => (
-                                <Table.Cell key={cell.id}>
-                                  {flexRender(
-                                    cell.column.columnDef.cell,
-                                    cell.getContext()
-                                  )}
-                                </Table.Cell>
-                              ))}
-                            </Table.Row>
-                          ))}
-                        </Table.Body>
-                      </Table>
-                      <LocalizedTablePagination
-                        canNextPage={table.getCanNextPage()}
-                        canPreviousPage={table.getCanPreviousPage()}
-                        nextPage={table.nextPage}
-                        previousPage={table.previousPage}
-                        count={count}
-                        pageIndex={table.getState().pagination.pageIndex}
-                        pageCount={table.getPageCount()}
-                        pageSize={PAGE_SIZE}
-                      />
-                    </div>
+                    <DataTable
```

Comment:
thought: Doesn't the DataTable handle the case of `noRecords` already? I think we can remove it from here to simplify the component.

Reply:
Good point - removed!